### ISSUE TITLE
QIO Item Deletion

### ISSUE THREAD

**Issue by anzu81** [mekanism/Mekanism#7655]
### Issue description

Everyone with a QIO system on the pack ATM 8 public server has lost hundreds of thousands if not millions of items from their QIO systems at least 2 of the user have the Super massive drive. 

### Minecraft version

1.19.2 (Latest)

### Forge version

43.1.47

### Mekanism version

10.3.5 (Latest)

### Other relevant versions

Mekanism-1.19.2-10.3.5.473

**Comment by Jorue23** [comment]
Maybe a bit more Info? Did you update your pack version? Did the server crash and afterwards it was like that, if yes can you send the crash log? Did it happen out of the blue? And also keep in mind that atm8 is still in beta and shouldn't be used publicly cause stuff like that is expected.

**Comment by pupnewfster** [comment]
Yeah, definitely going to need more information on this one. Like situations surrounding this happening, what sorts of items is it happening with, etc.
